src/robot.py

```diff
@@ -0,0 +1,103 @@
+import wpilib
+import wpilib.deployinfo
+
+import drivestation
+import driveteam
+
+import swerve.swervesubsystem
+import utils.utils
+
+from shuffleboard import addDeployArtifacts
+
+
+class MyRobot(wpilib.TimedRobot):
+  def __init__(self):
+    wpilib._wpilib.TimedRobot.__init__(self)
+
+    self.pilots = driveteam.DriveTeam()
+    self.drivebase = swerve.swervesubsystem.SwerveSubsystem()
+    self.strafe = 0.0
+    self.turn = 0.0
+    self.drive = 0.0
+    self.winch = False
+    self.hookextend = False
+    self.hookretract = False
+    self.hookleft = False
+    self.hookright = False
+
+    self.aim = 0.0
+    self.fire = 0.0
+    self.unjam = False
+    self.pickup = 0.0
+    self.eject = False
+
+  def getInputs(self):
+    self.strafe = self.pilots.get_strafe_command()
+    self.turn = self.pilots.get_turn_command()
+    self.drive = self.pilots.get_drive_command()
+    self.winch = self.pilots.get_winch_command()
+    self.hookextend = self.pilots.get_hook_extension_command()
+    self.hookretract = self.pilots.get_hook_retract_command()
+    self.hookleft = self.pilots.get_hook_left_command()
+    self.hookright = self.pilots.get_hook_right_command()
+
+    self.aim = self.pilots.get_aim_command()
+    self.fire = self.pilots.get_firing_command()
+    self.unjam = self.pilots.get_unjam_command()
+    self.pickup = self.pilots.get_pickup_command()
+    self.eject = self.pilots.get_eject_command()
+
+    self.togglefieldoriented = self.pilots.get_view_command()
+
```

Comment:
I'm a little opposed to creating variables to hold the return value of getter functions instead of just using the getter whenever you need it.

Reply:
Sampling the inputs at a fixed rate is fairly standard procedure for a digital control system, which is pretty much what the iterative robot is. We aren't going to be doing any modeling or characterizations but it at least avoids potential inconsistencies in differences between one sample of the controllers to another within the same iteration.